test/integration/puppetlabs/services/jruby_pool_manager/jruby_internal_test.clj

```diff
@@ -50,9 +52,12 @@
           (is (= CompatVersion/RUBY1_9 (.getCompatVersion container))
               "Unexpected default compat version configured for JRuby 1.7 container"))
         (finally
-          (.terminate container)))
+          (.terminate container)
+          (.terminate container-two)))
       ;; Because we add the current datetime to the filename we need to glob for it here.
```

Comment:
might want to update the comment to note the part about the hashcode too, since that will also be in the glob

Reply:
good call.

Reply:
updated.